kodilla-testing2/src/test/java/testing2/crudapp/CrudAppTestSuite.java

```diff
@@ -0,0 +1,132 @@
+        driver.navigate().refresh();
+
+        while (!driver.findElement(By.xpath("//select[1]")).isDisplayed()) ;
+
+        driver.findElements(By.xpath("//form[@class=\"datatable__row\"]")).stream()
+                .filter(anyForm ->
+                        anyForm.findElement(By.xpath(".//p[@class=\"datatable__field-value\"]"))
+                                .getText().equals(taskName))
+                .forEach(theForm -> {
+                    WebElement selectElement = theForm.findElement(By.xpath(".//select[1]"));
+                    Select select = new Select(selectElement);
+                    select.selectByIndex(1);
+
+                    WebElement buttonCreateCard =
+                            theForm.findElement(By.xpath(".//button[contains(@class, \"card-creation\")]"));
+                    buttonCreateCard.click();
+                });
+        Thread.sleep(5000);
+    }
+
+    public boolean checkTaskExistsInTrello(String taskName) throws InterruptedException {
+        final String TRELLO_URL = "https://trello.com/login";
+        boolean result = false;
+        WebDriver driverTrello = WebDriverConfig.getDriver(WebDriverConfig.CHROME);
+        driverTrello.get(TRELLO_URL);
+
+        driverTrello.findElement(By.id("user")).sendKeys("xxx");
+        driverTrello.findElement(By.id("password")).sendKeys("xxx");
+        driverTrello.findElement(By.id("login")).submit();
+
+        Thread.sleep(2000);
+
+        driverTrello.findElements(By.xpath("//a[@class=\"board-tile\"]")).stream()
+                .filter(aHref -> aHref.findElements(By.xpath(".//span[@title=\"Kodilla Application\"]")).size() > 0)
+                .forEach(aHref -> aHref.click());
+
+        Thread.sleep(2000);
+
+        result = driverTrello.findElements(By.xpath("//span")).stream()
+                .filter(theSpan -> theSpan.getText().contains(taskName))
+                .collect(Collectors.toList())
+                .size() > 0;
+
+        driverTrello.close();
+
+        return result;
+    }
+
+    public void testTaskCleaner(String taskName) throws InterruptedException{
+        while(!driver.findElement(By.xpath("//select[1]")).isDisplayed());
+
+        Thread.sleep(2000);
+
+        driver.findElements(By.xpath("//form[@class=\"datatable__row\"]")).stream()
+                .filter(anyForm ->
+                        anyForm.findElement(By.xpath(".//p[@class=\"datatable__field-value\"]"))
+                                .getText().equals(taskName))
+                .forEach(theForm -> {
+                    WebElement buttonDeleteTask =
+                            theForm.findElement(By.xpath(".//button[contains(., 'Delete')]"));
+                    buttonDeleteTask.click();
+                });
+                Thread.sleep(5000);
+    }
+
+    @Test
+    public void shouldCreateTrelloCard() throws InterruptedException {
+        String taskName = createCrudAppTestTask();
```

Comment:
it will be better, to have this test on top of this file, and all helper methods below of it.